realm/realm-library/src/main/cpp/io_realm_internal_Collection.cpp

```
@@ -414,8 +414,46 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_Collection_nativeDeleteFirst(J
     return JNI_FALSE;
 }
 
-JNIEXPORT void JNICALL Java_io_realm_internal_Collection_nativeDelete(JNIEnv* env, jclass, jlong native_ptr,
-                                                                      jlong index)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
```

We shouldn't hide this error in general, but it is fine while prototyping.